content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,119 @@
+
+## NetKVM VirtIO bringup
+NetKVM VirtIO is an NDIS 5 compatible driver developed by Red Hat Inc., which allows fast VirtIO networking in virtual machine software such as QEMU and other KVM hardware accelerated virtual machines.
+Nguyen Trung Khanh, a ReactOS contributor, took over Benjamin Aerni's work of importing the driver into the source tree of ReactOS.
+This provides ReactOS with support for another network interface out of the box.
+
+## Miscellaneous changes & improvements
+Other areas of ReactOS infrastructure have received lots improvements, ranging from device drivers to software applications of ReactOS by various developers and contributors. Notably FreeLoader, the ReactOS' bootloader, received incremental fixes and further support improvements. As such Hermès Bélusca-Maïto added support for booting Linux 64-bit systems in 64-bit FreeLoader and fixed an issue where
+FreeLoader couldn't read from an EXT volume hence preventing booting. Dmitry Borisov fixed a serious triple fault bug when ReactOS was booted up in Screen mode. In addition to that, Dmitry Borisov also provided other fixes as well as adding ARC-emulation support necessary for NEC PC-98 series.
+Mark Harmstone also brought some fixes for the bootloader so that Windows Vista can be properly booted. On-Screen Keyboard and Accessbility Utility Manager are being regularly updated by Bisoc George and at the same time he provided some fixes or minor improvements to other parts of ReactOS.
+
+Hervé Poussineau worked hard on ISA PnP (a driver component of PnP) which detects ISA devices although such driver is currently deactivated.
+Mark Jansen fixed a bug in comctl32 (Common Controls) module where Visual Basic 6 installer was rendered incorrectly. Eric Kohl worked on "Safely Remove Hardware" dialog manager where the peripherals are now enumerated for safely removal.
+
+What is particularly different between 0.4.13 and this release is the sleek performance due to obsolete fonts removed in ReactOS by Katayama Hirofumi MZ. This both reduced the binary size of this release and the RAM consumption.
+Here is a comparison between 0.4.13 (with the obsolete fonts used and loaded in memory) and 0.4.14.
+
+{{< figure link="/img/project-news/reactos-0414-released/misc.png" src="/img/project-news/reactos-0414-released/misc.png" >}}
+
+The 64-bit ReactOS port is extensively being worked on by Timo Kreuzer with continuous work on interrupt handling and other hardware issues like USB and networking issues in x64 build.
+Stanislav Motylkov still continues on working on Xbox boot support ranging from fixes patches and other updates brought by him. In addition to that Hermès Bélusca-Maïto fixed some old bugs, notably a corruption in the console emulation (CONSRV) layer's memory when converting
+input events back to ANSI and a bug in HAL where the system lead into an assertion failure sporadically.
```

Comment:
```suggestion
```

Reply:
All that section should go away? I've personally not verified when the CONSRV thing has been committed and whether it has been ported back into the 0.4.14 branch or not.

Reply:
It has been moved a bit

Reply:
It seems you accidentally removed me from the article with this suggestion... :sweat_smile: